### DISCUSSION TITLE
What is the `display` key in the default TS Config generated by `turbo-create`?

### DISCUSSION THREAD

**Discussion by Sensanaty** [vercel/turborepo#9221] [Help] [answered]
### Summary

When you run `npx turbo-create@latest`, it'll generate the following config file:

`packages/typescript-config/base.json`
```json
{
  "$schema": "https://json.schemastore.org/tsconfig",
  "display": "Default",
}
```

I'm wondering what that `display` key does? I can't find any documentation on this anywhere, my IDE doesn't tell me anything about it either, yet I see it in all the configs provided by turbo.

I assume it's something that is displayed when hosted/in CI?

**Comment by anthonyshew** [comment]
Interesting, I don't see anything documenting it either. It must be a mistake?

Do you want to make a PR to remove it? Otherwise, I will. 😄

**Reply by Sensanaty** [reply]
I don't mind making a PR for this, but there seems to be too many for it to be an accident?

My IDE is showing 56 results for `"display":`, so every single turborepo example `tsconfig` contains one of these keys. It must've been somewhat on purpose? It's hard for me to find since I'm not too familiar with the codebase and `display` is a generic enough word that it pops up everywhere, though.

But hey you're the maintainer here, so here's the PR :D https://github.com/vercel/turborepo/pull/9225

Let me know if I need to edit the PR in any way, first time contributing to the project so not sure if I missed anything (it's early and I haven't had my coffee yet :) )

**Reply by anthonyshew** [reply, marked as answer]
If I were to guess, it was the kind of thing that either got copy-pasted or "Well, it must be in there for a reason so I guess I'll keep it, too...?" 😄

Thanks for the PR! We'll move over there.
